src/Microsoft.ML.StandardLearners/Standard/Online/AveragedLinear.cs

```diff
@@ -17,38 +17,89 @@ namespace Microsoft.ML.Trainers.Online
 {
     public abstract class AveragedLinearArguments : OnlineLinearArguments
```

Comment:
>AveragedLinearArguments [](start = 26, length = 23)

xml

Reply:
added

---
In reply to: [256251939](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256251939) [](ancestors = 256251939)